Test/FMI3/fmi3_import_model_structure_test.c

```diff
@@ -92,7 +92,43 @@ static int test_parse_xml(int expectFailure, char* xmlTestRootDir, char* xmlDirR
 static void logger_invalid_state_derivative_reference(jm_callbacks* cb, jm_string module,
         jm_log_level_enu_t log_level, jm_string message)
 {
-    char* expMsg = "The state derivative 'state_var' does not specify the state variable that it is a derivative of.";
+    char* expMsg = "The continuous state derivative 'state_var' does not specify the state variable that it is a derivative of.";
+    if (!strncmp(expMsg, message, strlen(expMsg))) {
+        g_logger_found_err_msg = 1;
+    }
+
+    jm_default_logger(cb, module, log_level, message);
+}
+
+/* Tests that an error is raised when a ModelStructure list is incorrectly ordered. */
+static void logger_incorrect_order(jm_callbacks* cb, jm_string module,
+        jm_log_level_enu_t log_level, jm_string message)
+{
+    char* expMsg = "Parse error at line";
+    if (!strncmp(expMsg, message, strlen(expMsg))) {
+        g_logger_found_err_msg = 1;
+    }
+
+    jm_default_logger(cb, module, log_level, message);
+}
```

Comment:
If you rebase after #44 you can use fmi_testutil functions to find any log message.